ukaea/ditto
# Testing/system tests #18

Main changes:

* `systemTests/execution_space` is synced to make debugging easier. A simple readme file in the execution_space directory makes sure that it always exists when the repository is cloned.
* The DITTO Web API port is now set as part of the application configuration
* `/opt/minio/data` on the Dev Env VM is now readable by other processes: specifically, by the system tests running in python
* A system test checks that when a request to create bucket is sent, the corresponding directory appears in `/opt/minio/data`
* The system tests clear out the relevant directories in `/opt/minio/data` before they are run each time
* `systemTests/venv` no longer commited
* Minio logs to `/tmp/minio.log`


To test:

* reload and provision vagrant machine, ssh in
* `cd systemTests`
* `source venv/bin/activate`
* `pytest testScenarios`

`test_template.py` should pass
`test_template2.py` should fail

Changing `self.then.thing_is_false()` to `self.then.thing_is_true()` in `test_template2.py` makes it all pass
